projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts

```diff
@@ -1,29 +1,32 @@
-import { Component, EventEmitter, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
+import { Component, EventEmitter, Inject, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
```

Comment:
why do we have inject here

Reply:
That is a mistake I forgot to remove it.